concepts/nums-strs/about.md

```diff
@@ -0,0 +1,82 @@
+# Introduction
+
+AWK is a very small language and it just has a basic set of data types.
+In this lession we'll go over numbers and strings.
+
+## Numbers
+
+Numbers in AWK are implemented double-precision floating point numbers.
+There is no distinction between integers and floats.
+
+AWK understands octal number constants (starting with `0`) and hexadecimal number constants (starting with `0x` or `0X`).
+
+### Numeric operations
+
+You can use the [standard arithmetic operators][ops-arith] `+`, `-`, `*`, `/`, `%`.
+Both `^` and `**` can be used for exponentiation, but because `^` is specified by POSIX it is preferred.
+There are some [builtin numeric functions][funcs-numeric], including trigonometric functions, random number generation, and `int(x)` to truncate a number to the nearest integer between _x_ and zero.
+
+## Strings
+
+AWK strings are composed of ASCII bytes.
```

Comment:
gawk seems fine with UTF chars

Reply:
Indeed
```sh
$ echo "Hi friend😀!" | gawk '{print substr($NF, 7, 2)}'
😀!
```

... Ah, I see I misinterpreted this from the manual:
> A string constant consists of a sequence of characters enclosed in double quotation marks. For example:
> "parrot"
> represents the string whose contents are ‘parrot’. Strings in gawk can be of any length, and they can contain any of the possible eight-bit ASCII characters, including ASCII NUL (character code zero). Other awk implementations may have difficulty with some character codes.